.markdownlint.yml

```diff
@@ -0,0 +1,18 @@
+# markdownlint configuration.
+
+# For more information, see:
+# https://github.com/DavidAnson/markdownlint#optionsconfig.
+
+# Enable all rules.
+default: true
+
+line-length:
+  # Line length checking is not strict by default.
+  strict: true
+  line_length: 80
```

Comment:
Any specific reason for 80?

Reply:
Nothing in particular, 80 is something I'm most used to.